test/hts-precompile/token-transfer/tokenTransferContract.js

```diff
@@ -27,7 +27,7 @@ const {
   pollForNewSignerBalanceUsingProvider,
 } = require('../../../utils/helpers');
 
-describe('TokenTransferContract Test Suite', function () {
+describe('TokenTransferContract Test Suite', async function () {
```

Comment:
Why do we need `async` here?

Reply:
Hmm I see. I think we should leave the `async` keyword out and ignore the warning for now then could you please create a ticket to address this issue so we can come back and figure it out later?